jobs/cluster/templates/config/redis.conf

```diff
@@ -16,12 +16,37 @@ appendonly yes
 # we should already be *in* the store dir
 appendfilename "redis.aof"
 appendfsync everysec
+<% if p('redis.no-appendfsync-on-rewrite') %>
+no-appendfsync-on-rewrite "<%= p('redis.no-appendfsync-on-rewrite') %>"
+<% end %>
```

Comment:
once this is changed to a boolean, simply:
```
no-appendfsync-on-rewrite <%= p('redis.no-appendfsync-on-rewrite') %> ? "yes" : "no" %>
```

notably, also drop the erb value quotes.